pom.xml

```diff
@@ -259,6 +259,19 @@
                     <target>${maven.compiler.target}</target>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Why? We already have a way of running the benchmarks without packaging an uber jar

Reply:
I'd prefer a PR to just address the title of the PR, leave other concerns for other PRs.

Reply:
This was added via a much older version and we needed a jar to include into our project. Not needed in this PR.